src/nwbinspector/_nwb_inspection.py

```diff
@@ -319,7 +323,9 @@ def _intercept_in_vitro_protein(nwbfile_object: pynwb.NWBFile, checks: Optional[
         "check_subject_proper_age_range",
     ]
     subject_related_dandi_requirements = [
-        check.importance == Importance.CRITICAL for check in checks if check.__name__ in subject_related_check_names
+        check.importance == Importance.CRITICAL  # type: ignore
+        for check in checks
+        if check.__name__ in subject_related_check_names
```

Comment:
Whenever you see skips like this, it really is indicative of how we probably never should have attached attributes to a function to begin with

We've discussed how that decision caused other issues (and all the copy function nonsense to theoretically solve it) so long term, if the design of the registry is refactored to be more appropriate, these ignores should go away